### DISCUSSION TITLE
3.4.0rc1

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1136] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.4.0rc1'>3.4.0rc1</a>.</em>

**Comment by snowman2** [comment]
The plan is to release with PROJ 9.1 in the wheels.

Release Candidate Status:

- [ ] Linux ARM wheels
- [ ] MacOS ARM wheels (@vot4anto)
- [ ] Conda Forge
- [ ] Debian (@sebastic)
- [ ] Fedora (@jdkloe)

Plan to release over the weekend. Testing is always appreciated, but definitely not required.

**Reply by sebastic** [reply]
`setup.cfg` needs to be reinstated to fix the Debian package build as discussed in [#1134](https://github.com/pyproj4/pyproj/discussions/1134#discussioncomment-3591451).

**Reply by snowman2** [reply]
Switching back to setup.cfg for this release: https://github.com/pyproj4/pyproj/pull/1137
